src/Data/Container/Indexed/Relation/Binary/Equality/Setoid.agda

```diff
@@ -0,0 +1,59 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Equality over indexed container extensions parametrised by a setoid
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Relation.Binary using (Setoid)
+
+module Data.Container.Indexed.Relation.Binary.Equality.Setoid
+  {ℓⁱ ℓᶜ ℓᵉ} {I : Set ℓⁱ} (S : I → Setoid ℓᶜ ℓᵉ)
```

Comment:
I think this should be `IndexedSetoid` from `Relation.Binary.Indexed.Homogeneous(.Bundles)?`

Reply:
What is the advantage of packing this as `IndexedSetoid`?  
Note that `Func` does not exists for `IndexedSetoid`.  With
`S T : I -> Setoid` I can get a function as `(i : I) -> Func (S i) (T i)`, but there seems nothing as convenient for `S T : IndexedSetoid`, as far as I could see.

Reply:
Hmm from my perspective, the first advantage is that we keep a consistent interface in the library, the second is that `IndexedSetoid` is a record object that you can open and get everything useful you want immediately (with the right implicits/explicits). In contrast `I -> Setoid _ _` is difficult to access the individual indexed proofs.

 Yup, `IndexedFunc` doesn't exist yet but that doesn't mean it shouldn't exist in the future.